pkg/model/bootstrapscript.go

```diff
@@ -133,6 +142,24 @@ func (b *BootstrapScript) ResourceNodeUp(ig *kops.InstanceGroup, cs *kops.Cluste
 	return fi.WrapResource(templateResource), nil
 }
 
+// getRelevantHooks returns a list of hooks to be applied to the instance group
+func (b *BootstrapScript) getRelevantHooks(hooks []kops.HookSpec, role kops.InstanceGroupRole) []kops.HookSpec {
+	relevantHooks := []kops.HookSpec{}
+	for _, hook := range hooks {
+		if len(hook.Roles) == 0 {
```

Comment:
What happens with an image where docker or the proper docker is not installed?  For instance, RHEL does not have docker installed?